google-cloud-storage/src/main/java/com/google/cloud/storage/Bucket.java

```diff
@@ -1114,6 +1114,45 @@ public boolean deleteDefaultAcl(Entity entity) {
     return storage.deleteDefaultAcl(getName(), entity);
   }
 
+  /**
+   * Deletes the lifecycle rules of this bucket.
+   *
+   * <p>Example of deleting the lifecycle rules of this bucket.
+   *
+   * <pre>{@code
+   * ImmutableList<BucketInfo.LifecycleRule> lifecycleRules =
+   *   ImmutableList.of(
+   * 	  new BucketInfo.LifecycleRule(
+   * 		  BucketInfo.LifecycleRule.LifecycleAction.newSetStorageClassAction(
+   * 			  StorageClass.COLDLINE),
+   * 		  BucketInfo.LifecycleRule.LifecycleCondition.newBuilder()
+   * 			  .setAge(1)
+   * 			  .setNumberOfNewerVersions(3)
+   * 			  .setIsLive(false)
+   * 			  .setCreatedBefore(new DateTime(System.currentTimeMillis()))
+   * 			  .setMatchesStorageClass(ImmutableList.of(StorageClass.COLDLINE))
+   * 			  .build()),
+   * 	  new BucketInfo.LifecycleRule(
+   * 		  BucketInfo.LifecycleRule.LifecycleAction.newDeleteAction(),
+   * 		  BucketInfo.LifecycleRule.LifecycleCondition.newBuilder().setAge(1).build()));
+   * Bucket bucket =
+   *   storage.create(
+   * 	  BucketInfo.newBuilder(bucketName)
+   * 		  .setLocation("us")
+   * 		  .setLifecycleRules(lifecycleRules)
+   * 		  .build());
+   * Map<LifecycleRule, Boolean> results =
+   *   bucket.deleteLifecycleRules(lifecycleRules.get(0));
+   * }</pre>
+   *
+   * @param rules the set of OLM rules to delete
+   * @return the OLM rules and their results
+   * @throws StorageException upon failure
```

Comment:
I have no idea what is 'OLM'

Reply:
OLM stands for `object lifecycle management` I have just removed it and make it only `lifecycle` to be more readable.